src/base/ddl_parser_test.cc

```diff
@@ -385,19 +396,18 @@ TEST_F(DDLParserTest, joinExtract) {
         LOG(INFO) << "after add index:\n" << DDLParser::PhysicalPlan(sql, db);
     }
 
-    // TODO: fix later
-    // {
-    //     ClearAllIndex();
-    //     // left join
-    //     auto sql = "SELECT t1.col1, t1.col2, t2.col1, t2.col2 FROM t1 left join t2 on t1.col1 = t2.col2;";
-    //
-    //     auto index_map = ExtractIndexesWithSingleDB(sql, db);
-    //     // {t2[col_name: "col2" ttl { ttl_type: kLatestTime lat_ttl: 1 }, ]}
-    //     CheckEqual(index_map, {{"t2", {"col2;;lat,0,1"}}});
-    //     // the added index only has key, no ts
-    //     AddIndexToDB(index_map, &db);
-    //     LOG(INFO) << "after add index:\n" << DDLParser::PhysicalPlan(sql, db);
-    // }
+    {
+        ClearAllIndex();
+        // left join
+        auto sql = "SELECT t1.col1, t1.col2, t2.col1, t2.col2 FROM t1 left join t2 on t1.col1 = t2.col2;";
+
+        auto index_map = ExtractIndexesWithSingleDB(sql, db);
+        // {t2[col_name: "col2" ttl { ttl_type: kLatestTime lat_ttl: 1 }, ]}
+        CheckEqual(index_map, {{"t2", {"col2;;lat,0,1"}}});
```

Comment:
should `lat,0,0`

Reply:
what we need in t2 is the last one of t2 where col2 == t1.col1, so it should be lat 1?